README.md

```diff
@@ -396,7 +391,31 @@ You're done! Congratulations!
 3. The key will change if you get a new bumper because the bumper has distance sensors that use the security key. Instead of applying the existing key to the bumper, they replace the key on all parts of the car. The same goes for many other parts with SecOC components. Even if you never get into an accident, the key can still change if a Toyota service technician presses a wrong button.
 </details>
 
-### Step 7. What's next?
+### Step 7. Clean up
+
+Put the covers back on, and you're done. Congratulations!
+
+Comma Power (OBD2 connector + long cable) is optional. It's not necessary for using C3X, but keeping it allows C3X to stay powered on when you turn off the car.
+
+<details><summary>Comma Power</summary>
+
+__Pros__
```

Comment:
Didn't add dashcam benefits because I don't know anything about it. Suggestions welcome.

Reply:
Hm are those applicable to car-off scenarios?

Reply:
Hmm, I see. It's convenient for retrieving them but not necessary.

Reply:
I do have a section about easy SSH. I'll try to add this to that bullet point